addons/bestja_account_deletion/models.py

```diff
@@ -21,7 +21,7 @@ class Volunteer(models.Model):
 
     @api.one
     def delete_account(self):
-        self.write({'login': uuid.uuid1(),
+        self.sudo().write({'login': uuid.uuid1(),
```

Comment:
Since we’re using `sudo`, shouldn’t we add [some permission checking](https://github.com/EE/bestja/blob/68132d11d9e7f374a39816e2e7d9588fe664164e/addons/bestja_stores/models.py#L341) in that method?